### PR TITLE
Fixed issue #188

### PR DESCRIPTION
**Pull request by anshulwadhawan**
@mariobehling 
Changed the image's dimensions as required by replacing the old image.

### CODE REVIEW

File: team.html [hunk 1]
```diff
@@ -1132,12 +1132,11 @@ <h1 class="text-white">FOSSASIA Teams</h1>
 									<a href="https://www.facebook.com/techilicious.tarun"><i class="icon social_facebook"></i></a>
 									<a href="https://twitter.com/TarunKumar100"><i class="icon social_twitter"></i></a>
 									<a href="https://www.linkedin.com/in/meets2tarun"><i class="icon social_linkedin"></i></a>
-									<a href="https://sucode.me"><i class="icon social_blogger"></i></a>
```
Comment (isuruAb) [inline team.html]: hello @anshulwadhawan why did you delete this line? please explain me 👍

Comment (anshulwadhawan) [reply team.html]: Hello @isuruAb 
These were some IDE issues.Please review as I have done the corrections.Now only the image has been changed by dimensions.No edits have been done to the code.

**Comment by mariobehling** [comment]
Image size should be much smaller.
